modules/ROOT/pages/runtime-manager-agent-notifications.adoc

```diff
@@ -9,23 +9,34 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid
 image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-The Anypoint Runtime Manager agent publishes notifications, in JSON format, about events that occur in the Mule instance. This allows you to implement your own system for receiving and handling notifications. Notifications are sent over both the REST and WebSocket transports (for more information on transports refer to the Runtime Manager agent architecture documentation).
+The Anypoint Runtime Manager agent publishes notifications, in JSON format, about events that occur in the Mule instance.
+This enables you to implement a system for receiving and handling notifications.
+Notifications are sent over both the REST and WebSocket transports.
+For more information about transports, see xref:runtime-manager-agent-architecture.adoc[Runtime Manager Agent Architecture].
 
-== Configure the Mule Agent Notifications Callback URL
+== Configure the Agent Notifications Callback URL
 
-To receive notifications on the specified URL, you must first configure the `callback` URL in the `mule-agent.yml` file inside the REST transport section as shown below.
+To forward Runtime Manager agent notifications to a URL over the REST transport, configure a `callback` URL in the mule-agent.yml file.
 
-[TIP]
-To enable notification to be sent, you must enable the `rest.agent.transport` section. Also ensure the configured URL ends with a trailing slash (`/`). This is required since the agent will append the path to it.
+To configure the `callback` URL:
 
+. Add the `callback` URL to the `mule-agent.yml` file:
++
 [source,yaml,linenums]
 ----
+transports:
   rest.agent.transport:
     enabled: true
     callback:
-      url: http://localhost:8080/
-    port: 8888
+      url: http://0.0.0.0:8081/test/
```

Comment:
Is there a reason for changing this example? I'm not sure is that configuration will work.

Reply:
I used the example from this KB:
https://help.mulesoft.com/s/article/How-to-configure-a-callback-URL-to-forward-agent-notifications-in-mule-runtime

Reply:
this is incorrect - the correct value was the previous one.

Reply:
Reverted.